src/core/lib/channel/channelz_registry.cc

```diff
@@ -62,7 +62,7 @@ void ChannelzRegistry::InternalRegister(BaseNode* node) {
 }
 
 void ChannelzRegistry::MaybePerformCompactionLocked() {
-  constexpr double kEmptinessTheshold = 1 / 3;
+  constexpr double kEmptinessTheshold = 1. / 3;
```

Comment:
In order to avoid such mistakes, it's better to use initialization 
```suggestion
  constexpr double kEmptinessTheshold{1. / 3};
```

Reply:
Sorry I'm probably missing something. How would uniform initialization save us from that? The following snippet compiles without any warning and will set the value to `0`:
```
double kEmptinessTheshold{1/3};
```